### PR TITLE
Upgrade from Chromium 75.0.3770.75 to Chromium 75.0.3770.80 (Uplift to 0.66.x)

### PR DESCRIPTION
**Pull request by bsclifton**
Related https://github.com/brave/brave-core/pull/2600
Uplift of https://github.com/brave/brave-browser/pull/4718
Fixes https://github.com/brave/brave-browser/issues/4724

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "0.66.x",
+        "branch": "c75bump-0.66.x",
```
Comment (bbondy) [inline package.json]: why the branch change here?

**Comment by mkarolin** [comment]
CI succeeded (except for unrelated lint errors): https://staging.ci.brave.com/job/brave-browser-build-pr/job/PR-4726/2/flowGraphTable/

File: package.json (hunk 1 above)
Comment (mkarolin) [reply package.json]: This was for CI. Removed the commit with this changed.

Comment (mihaiplesa) [reply package.json]: explicit pinning won't be needed anymore as I'll uplift my pipeline changes today

**Pull request merged**